Calculator_exercise.py

```diff
@@ -0,0 +1,299 @@
+"""Calculator exercise"""
```

Comment:
You pushed this to "solved matrix exercise 1." I assume this is meant to be in its own branch (I'm not sure how you were told to do things).